README.md

```diff
@@ -82,76 +134,150 @@ Asks the user for a shared web credential. Requires additional setup both in the
 
 Sets a shared web credential. Resolves to `true` when successful.
 
-### `canImplyAuthentication([{ authenticationType }])`
+### `canImplyAuthentication([{ authenticationType }])` (iOS only)
 
 Inquire if the type of local authentication policy is supported on this device with the device settings the user chose. Should be used in combination with `accessControl` option in the setter functions. Resolves to `true` if supported.
 
-### `getSupportedBiometryType()`
+### `getSupportedBiometryType([{}])`
 
 Get what type of hardware biometry support the device has. Resolves to a `Keychain.BIOMETRY_TYPE` value when supported, otherwise `null`.
+
 > This method returns `null`, if the device haven't enrolled into fingerprint/FaceId. Even though it has hardware for it.
 
-### `getSecurityLevel()` (Android only)
+### `getSecurityLevel([{}])` (Android only)
 
 Get security level that is supported on the current device with the current OS.
 
-### Security Levels (Android only)
+### Options
 
-If set, `securityLevel` parameter specifies minimum security level that the encryption key storage should guarantee for storing credentials to succeed.
+| Key                        | Platform     | Description                                                                                      | Default                                                      |
+| -------------------------- | ------------ | ------------------------------------------------------------------------------------------------ | ------------------------------------------------------------ |
+| **`accessControl`**        | All          | This dictates how a keychain item may be used, see possible values in `Keychain.ACCESS_CONTROL`. | *None* (iOS), `BIOMETRY_ANY` default for Android.            |
```

Comment:
I understand you well, but that will not make this PR merged in next few hours. That will delay everything for ages... 

so let's merge this first and then work on next PRs. This PR is too large

Reply:
Hmm, the feature has been under slow progress for a year and a half now. Although the progress you have made recently is amazing, I'm afraid it might be hard to get it reviewed and approved within a matter of hours.

Personally, I believe the large scope of the PR is what is might makes the progress so slow. Limiting the changes to as few as possible would possibly make the PR progress faster.

Reply:
Large scope of PR is due to global refactoring that is needed for making biometrics works. 

Its a major update of the functionality. Limiting changes approach if possible will be used, but for making this functionality work - required all those changes. (we should do a trade-off between completeness of changes and number of changes).

Reply:
Unfortunately, I'm not iOS developer, so updates in iOS code are quite hard for me to do quickly... any help appreciated.